code/Chapter4/4_param_test_based_fixture.cpp

```diff
@@ -0,0 +1,75 @@
+#include <iostream>
```

Comment:
```suggestion
```

Reply:
Resolved in abab5e4457494945e7c2f0fca8e116aed1c8281e